tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -250,28 +295,16 @@
        this._change_list = runEnvironmentJson.p['change_list'];
        this._build_time = runEnvironmentJson.p['build_time'];
        this._build_target = runEnvironmentJson.p['build_target'];
-       this._statement = recommendationJson.p['statement'];
-
-       if (statement.includes('HIGHLY')) {
-         this.customStyle['--summary-color'] = 'red';
-       } else if (statement.includes('MODERATE')) {
-         this.customStyle['--summary-color'] = 'orange';
-       }
-
-       this.updateStyles();
-
-       this._showDeviceStepChart(inputAnalysisJson);
-       this._showTopOpsTable(generalAnalysisJson);
-       this._showRecommendation(recommendationJson);
      },
 
      /* Plots a graph of step time (in ms) against step number */
      _showDeviceStepChart : function(deviceJson) {
-       var chart = this.$$('#device_step_chart');
-       if (!deviceJson || !deviceJson.rows ||
+        if (!deviceJson || !deviceJson.rows ||
                !deviceJson.rows.length || !this._active) {
+         this._show_step_time_graph = false;
```

Comment:
it might be better to make this property computed based on above values.

Reply:
Done.